daemon/src/shared_protocol.rs

```diff
@@ -50,13 +52,12 @@ pub(crate) async fn verify_cets(
 
         anyhow::Ok(())
     })
-    .in_current_span()
     .await??;
 
     Ok(())
 }
 
-#[instrument(skip_all)]
+#[instrument(target = "verify_crypto", level = "trace", skip_all)]
```

Comment:
IMHO I would probably call them like module names, `shared_protocol` here, and `contract_setup` in the other case (no need to append the `_deprecated` there.

I'm not too fussed though, I'm happy as long as they're not printed out by default :)

Reply:
fair enough. I was mostly pointing out that `shared_protocol` would not work for both, and this seemed to be the name liked by @luckysori most. As said above, I'm not too fussed; `verify_crypto` is a decent name too 👍

Reply:
I think `verify_crypto` is also fine!